website/static/js/filepage/index.js

```diff
@@ -198,12 +199,17 @@ var FileViewPage = {
         $('.file-view-panels').removeClass().addClass('file-view-panels').addClass(fileViewPanelsLayout);
 
         m.render(document.getElementById('toggleBar'), m('.btn-toolbar[style=margin-top:20px]', [
-            ctrl.canEdit() ? m('.btn-group', {style: 'margin-left: 0;'}, [
-                m('.btn.btn-sm.btn-danger.file-delete', {onclick: $(document).trigger.bind($(document), 'fileviewpage:delete')}, 'Delete')
+            ctrl.sourceUrl !== 'None' ? m('.btn-group', {}, [
```

Comment:
`None` would be serialized as `null` in javascript not the string value `'None'`